src/utilities/ToolTipTimestamp.tsx

```diff
@@ -0,0 +1,35 @@
+import { Tooltip, Typography } from "@mui/material";
+import formatTimestamp from "./formatTimestamp";
+import formatToolTipTimestamp from "./formatToolTipTimestamp";
+
+type ToolTipTimestampProps = {
+  ToolTipTimestamp: string;
```

Comment:
Props should be camel case (ex. `tooltipTimestamp`). Also, can we change this name to something like `timestamp`?

Reply:
Done